sig-security/annual-report-2021.md

```diff
@@ -0,0 +1,81 @@
+    *   KEP [1981](https://github.com/kubernetes/enhancements/issues/1981) - Support for Windows privileged containers - has matured toward alpha, including SIG Security member feedback. Nice work, team!
+*   **What areas and/or subprojects does the group need the most help with?**
+    *   We always welcome help from anyone aligned with our vision of improving Kubernetes security by learning together, sharing our expertise, and encouraging cross-SIG collaboration. Ideally, we would like to eventually have participating members with crossover to every SIG, providing a natural conduit of ideas and forming a network of helpful hackers throughout Kubernetes.
+    *   As a newer SIG with a broad mission, our most pressing needs right now are around outreach and expertise.
+        *   Today, people can help broaden our outreach in a few concrete ways:
+            *   Look out for one another, so we can be safer together! If you see a Kubernetes PR or Issue (or Enhancement, or whatever!) that looks like it may be security-relevant, you can tag sig/security to bring them to our attention and get SIG Security input. You can also bring them to our meetings as a topic for discussion!
+            *   If you spot things happening in other SIGs that may be relevant, or if something that is relevant to security interests is happening in your own SIG, let us know! Post in #sig-security to get community attention on it, or DM the SIG Security chairs on Slack so we can help. 
+            *   We don’t always have to be involved; another way you can help is by reaching out directly to other SIGs whose work may be affected by or related to your own. We all benefit when we work together better, especially on security-related topics.
+            *   Come to our meetings and get more involved! If you have friends who are interested in Kubernetes security, encourage them to come and bring their energy and ideas too. We’d love to see you and hear what you have to say!
+        *   To bring additional expertise to the table and help balance the load for SIG Leads, we would like to grow new Leads in the future, and are determining how best to implement the Tech Lead role in a SIG like ours. We would like to grow one or two interested members into that new role, and will be looking for help with this bootstrapping process when that time comes.
+*   **What's the average open days of a PR and Issue in your group?**
+    *   As a SIG who doesn’t own code in a project that encourages non-code contributions, we find the question of which PRs and issues belong to us to be difficult to answer, and would like to encourage further inclusivity of different kinds of contribution in questions on future reports.
```

Comment:
I can relate as someone that's involved in a significant amount of non-code contributions. The way we (ContribEx) try and gain greater visibility into this is by moving AIs and tasks into issues as that is broadly what the community uses and the tooling supports. This is largely for greater discoverability, recognition and transparency (both as a project and for employers who look for this sort of thing 😬).

One potential thought, there are several SIGs that have their own dedicated repos (e.g. [sig-release](http://github.com/kubernetes/sig-release) and [sig-testing](http://github.com/kubernetes/sig-testing)) that serve as "meta" repos for documentation on processes and a place where they can create and assign issues.

I'd be happy to chat about potential options both GitHub centric and other ideas. :) We've explored a few that could bubble other metrics like slack/mailing list activity, but so far they all get into the creepy area of PII and lack of [consent](https://github.com/kubernetes/community/issues/5260#issuecomment-763037701) >_>